src/App.tsx

```diff
@@ -1,28 +1,55 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './components/api';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .then(() => setLoading(true))
+      .catch(() => {
+        setHasLoadingError(true);
+      });
+  }, []);
```

Comment:
Чому ти кажеш `setLoading(true)` після того як завантажила тудушкі?
```suggestion
  useEffect(() => {
  	setLoading(true);
  
    getTodos()
      .then(setTodos)
      .then(() => setLoading(false))
      .catch(() => {
        setHasLoadingError(true);
      });
  }, []);
```